### DISCUSSION TITLE
copypasta_windows-v1.2 (a lot of new features !!)

### DISCUSSION THREAD

**Discussion by ThaaoBlues** [CopyPastaOfficial/CopyPasta#10] [Q&A] [answered]
Update of CopyPasta that brings this software to a whole new level of file sharing !! All features are described on [the official website](https://copypastaofficial.github.io/)

<hr /><em>This discussion was created from the release <a href='https://github.com/CopyPastaOfficial/CopyPasta/releases/tag/1.2'>copypasta_windows-v1.2 (a lot of new features !!)</a>.</em>

**Comment by AntoninLoubiere** [comment, marked as answer]
Edit: wrong website (error 404): the website is https://copypastaofficial.github.io/

**Reply by ThaaoBlues** [reply]
Thanks a lot ! Edited it 👍
